pkg/runner/cluster_k8s.go

```diff
@@ -10,13 +10,17 @@ import (
 	"os"
```

Comment:
FYI this whole file is revamped at https://github.com/ipfs/testground/pull/433 - I think the fact that we got rid of `job` will also make it easier to start various compositions/images, and not have to manage m jobs of n pods.